src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/fields/Selection.js

```diff
@@ -188,6 +211,34 @@ class Selection extends React.Component<Props> {
         return type;
     }
 
+    @computed get autoCompleteIdProperty() {
```

Comment:
I think at some point I had multiple usages of that, but it seems some changes removed that again. Can remove it again.

Reply:
Found another usage of the `idProperty`, where I can also use this getter now. Therefore I am going to leave it, so that this getter is used twice now.

Reply:
alright 🙂